coretex/entities/project/project.py

```diff
@@ -33,9 +34,16 @@ class Project(BaseObject):
     """
 
     tasks: List[Task]
+    visibility: ProjectVisibility
 
     @classmethod
-    def createProject(cls, name: str, projectType: ProjectType, description: Optional[str] = None) -> Self:
+    def createProject(
+        cls,
+        name: str,
+        projectType: ProjectType,
+        description: Optional[str] = None,
+        visiblity: Optional[ProjectVisibility] = ProjectVisibility.private
```

Comment:
Why is `visibility` optional?

Reply:
do you agree removing visibility parameter completely? since it does nothing right now, every project will be private no matter what is passed here